content/docs/gke/customizing-gke.md

```diff
@@ -89,17 +89,13 @@ To use VMs with more CPUs or RAM:
 
 To grant additional users IAM permissions to access Kubeflow:
 
-  * Add users by editing [cluster-kubeflow.yaml](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/deployment/gke/deployment_manager_configs/cluster-kubeflow.yaml#L74).
-
-
-By default, this file will be located at `${KUBEFLOW_SRC}/kubeflow/gcp_config/cluster-kubeflow.yaml` after your first deployment. After making changes to the file, you'll need to apply them:
+  * Users can be granted access to Kubeflow by adding the “IAP-secured Web App User” role on the GCP console [here](https://console.cloud.google.com/iam-admin/iam). Make sure you are in the same project as your Kubeflow deployment.
 
+  * The update can be confirmed by inspecting the IAM policy for your project. This can be accessed by running:
```

Comment:
Passive -> active:

```
You can confirm the update by inspecting the IAM policy for your project:
```

(I've also removed the second sentence to shorten the instructions.)